core/domain/rule_domain.py

```diff
@@ -16,65 +16,13 @@
 
 """Classes relating to rules."""
```

Comment:
Is there an advantage to keeping rule_domain around? I feel as though we should just get rid of it.